### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by ManarOmari**
In this milestone I did the following:

- Created desktop version for Portfolio 🖥️ 
 *** Add media query for desktop 
 *** Update the Welcome page to be fit for both desktop 🖥️ and mobile 📱 versions.
 *** Update the Work sections to be fit for both desktop 🖥️ and mobile 📱 versions.
 *** Update the About me sections to be fit for both desktop 🖥️ and mobile 📱 versions.
 *** Update the Form sections to be fit for both desktop 🖥️ and mobile 📱 versions.
 *** Update the Footer to be fit for both desktop 🖥️ and mobile 📱 versions.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -420,3 +418,188 @@ footer {
...
+  }
+
+  .display {
+    display: none;
+  }
+
+  .show {
+    display: inline-flex;
+  }
+}
```
Comment (cynthiainga) [inline assets/css/style.css]: - [ ] Please make sure your design stick as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) 🙃 🙃
Your design 


Figma design
Screen Shot 2022-08-25 at 14 41 12

- [ ] Please kindly fix your mobile design for all sections 🙃 😉 

Your design


Figma
Screen Shot 2022-08-25 at 14 51 06

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @ManarOmari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] Linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css [hunk 2]
```diff
@@ -415,8 +413,216 @@ footer a {
...
+    display: flex;
+  }
+
+  .navListItem {
+    padding: 4%;
+  }
+
+  .navItems li a {
+    text-decoration: none;
+  }
```
Comment (levy002) [inline assets/css/style.css:461]: - [x] Kindly align the navbar links correctly exactly as how it looks on the Figma design. :wink: 
- [x] Kindly change the link's colors to look like the provided design. :100: 

   - ### Your design
   image


  - ### Figma design
    image

File: assets/css/style.css [hunk 3]
```diff
@@ -415,8 +413,216 @@ footer a {
...
+    color: white;
+  }
+
+  .display {
+    display: none;
+  }
+
+  .show {
+    display: block;
+  }
```
Comment (levy002) [inline assets/css/style.css:647]: - [x] Kindly align this `RECENT WORKS` title correctly. Please adjust your spacing and align it correctly to stick to the provided Figma design. :wink: 

   - ### Your design
   image


  - ### Figma design
   image

File: assets/css/style.css [hunk 4]
```diff
@@ -415,8 +413,216 @@ footer a {
...
+
+  .projectBTNs {
+    margin-left: 0;
+  }
+
+  .projectBTNs li button {
+    transition: 0.3s ease-in-out;
+    padding: 14px 4px;
+  }
```
Comment (levy002) [inline assets/css/style.css]: - [x] Kindly make sure to stick to the provided design by aligning this stack list correctly. :wink: 

   - ### Your design
   image


  - ### Figma design
   image

File: assets/css/style.css [hunk 5]
```diff
@@ -415,8 +413,216 @@ footer a {
...
+
+  footer ul {
+    display: flex;
+    justify-content: center;
+  }
+
+  .seeProject {
+    transition: 0.3s;
+  }
```
Comment (levy002) [inline assets/css/style.css]: - [ ] Kindly adjust your button `border` styles to meet the provided design. :100: 

  - ### Your design
   image


  - ### Figma design
   image

File: assets/css/style.css [hunk 6]
```diff
@@ -415,8 +413,216 @@ footer a {
...
+
+  .aboutMe {
+    background-image: url("/assets/images/about-me-bg-left-desktop.svg"), url("/assets/images/about\ me-right-\ desktop.svg");
+    background-repeat: no-repeat, no-repeat;
+    background-size: auto, auto;
+  }
+
+  .aboutMe p {
+    margin-bottom: 2%;
+  }
```
Comment (levy002) [inline assets/css/style.css]: - [x] Kindly make sure to stick to the provided design. Please make sure to use the correct fonts and font size while styling the about-title and about-paragraph to look like the provided design. :100: 
- [x] Kindly make sure to place the about section background images correctly exactly how it looks on the Figma design. :wink: 

  - ### Your design
 
   image


  - ### Figma design
   image

File: index.html [hunk 7]
```diff
@@ -160,6 +178,7 @@ <h2 >About me</h2>
       <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hestiate to contact me.</p>
       <button class="seeProject inter-font" type="button">Get My Resume</button>
+      <div class="works workSkills">
       <ul class="skills">
```
Comment (levy002) [inline index.html:187]: - [x] Kindly make sure to adjust the skills card on the desktop version to look exactly as the provided Figma design. :wink: 

   - ### Your design
    image


  - ### Figma design
   image

File: assets/css/style.css [hunk 8]
```diff
@@ -415,8 +413,216 @@ footer a {
...
+
+  .form-section {
+    display: flex;
+    background-image: url("/assets/images/contact\ form-left-desktop.svg"), url("/assets/images/contact-form-bg-middle-desktop.svg"), url("/assets/images/contact-form-right-desktop.svg");
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position-x: right, center, left;
+    background-position-y: bottom, bottom, bottom;
+    background-size: 7%, 4%, 10%;
+  }
```
Comment (levy002) [inline assets/css/style.css:573]: - [ ] KIndly make sure you align your `form` section correctly by adjusting your spacing like margins, padding, and width. 
- [ ] Make sure every content inside has the correct width to look exactly as the provided Figma design. :100: 
- [ ] Kindly reduce the form paragraph `font-size` to meet the design :wink: 


   - ### Your design
   image


   - ### Figma design
   image

File: assets/css/style.css [hunk 9]
```diff
@@ -415,8 +413,216 @@ footer a {
...
+    display: flex;
+    justify-content: center;
+    justify-items: center;
+    flex-direction: column;
+  }
+
+  footer ul {
+    display: flex;
+    justify-content: center;
+  }
```
Comment (levy002) [inline assets/css/style.css]: - [ ] Kindly make sure to align all social media links correctly to look like the provided design. :wink: 

  - ### Your design
 
   image


  - ### Figma design
   image

File: assets/css/style.css [hunk 10]
```diff
@@ -415,8 +413,216 @@ footer a {
...
-  justify-items: center;
+.display {
+  display: block;
+}
+
+.show {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (levy002) [inline assets/css/style.css:424]: - [x] Kindly after finishing implementing all changes on the desktop version. Let's take a step back and make everything on our mobile version aligned correctly as on the provided Figma design. :wink: Please make sure that everything looks exactly like the provided design to make sure that our app is fully responsive :100:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @ManarOmari    ,  :wave: 
### I'm Levy, your PR reviewer

Great job so far :+1:  
However, they are some issues that you need to address before you move on to the next project :wink:

## Highlights
- Good job for using the correct Github flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Descriptive PR description. :heavy_check_mark: 
### Status: REQUESTED CHANGES :recycle: 
_check my comments under review_
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css (hunk 2 above)
Comment (ManarOmari) [reply assets/css/style.css:461]: change links color

Comment (ManarOmari) [reply assets/css/style.css:461]: change the margin and justify content

File: assets/css/style.css (hunk 3 above)
Comment (ManarOmari) [reply assets/css/style.css:647]: fix alignment of my recent work

File: assets/css/style.css (hunk 4 above)
Comment (ManarOmari) [reply assets/css/style.css]: update the padding of buttons

File: assets/css/style.css (hunk 5 above)
Comment (ManarOmari) [reply assets/css/style.css]: remove border buttons and update the width

File: assets/css/style.css (hunk 6 above)
Comment (ManarOmari) [reply assets/css/style.css]: - update font size and font family for about me 
- update background

File: index.html (hunk 7 above)
Comment (ManarOmari) [reply index.html:187]: update the design to adjust Figma

File: assets/css/style.css (hunk 9 above)
Comment (ManarOmari) [reply assets/css/style.css]: update the  social media space

File: assets/css/style.css (hunk 10 above)
Comment (ManarOmari) [reply assets/css/style.css:424]: fixed

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @ManarOmari , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented suggested changes.

### Optional suggestions :arrow_down_small: 

- [x] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**